gradle.properties

```diff
@@ -1,16 +1,18 @@
 org.gradle.jvmargs=-Xmx2G
 
-minecraft_version=1.19.4
-yarn_mappings=1.19.4+build.1
-loader_version=0.14.19
+minecraft_version=1.20.1
+yarn_mappings=1.20.1+build.1
+loader_version=0.14.21
 
-fabric_version=0.77.0+1.19.4
+fabric_version=0.83.0+1.20.1
 
-mod_version = 0.8+1.19.4
+mod_version = 0.8.1+1.20.1
 maven_group = foundationgames
 archives_base_name = enhancedblockentities
 
-arrp_version=0.6.7
+arrp_version=0.6.8
```

Comment:
Can't find that version of arrp where did you get it ?

Reply:
> Can't find that version of arrp where did you get it ?

https://github.com/Brokkonaut/ARRP

Reply:
The mod compiles and run with arrp 0.6.7

Reply:
@Brokkonaut can you make a PR for that version bump? Maybe that'll make the creator of ARRP do something.

Reply:
> @Brokkonaut can you make a PR for that version bump? Maybe that'll make the creator of ARRP do something.

Is was not necessary, I reverted that change. This PR now compiles against ARRP 0.6.7